features/user_profile/user-edits-profile.feature

```diff
@@ -81,3 +81,15 @@ Feature: As a registered user
     And I fill in t("devise.registrations.edit.current_password") with "password"
     And I click on t("devise.registrations.edit.submit_button_label") button
     And I should see t("hello", name: 'NewUlysses')
+
+  Scenario: Member edits profile
```

Comment:
clarify intent with more descriptive name: "Member edits contact email in profile"